docs/src/main/paradox/migration-guide/index.md

```diff
@@ -4,11 +4,7 @@
 
 @@@ index
 
-* [migration-guide-10.2.x](migration-guide-10.2.x.md)
-* [migration-guide-10.1.x](migration-guide-10.1.x.md)
-* [migration-guide-10.0.x](migration-guide-10.0.x.md)
-* [migration-guide-2.4.x-10.0.x](migration-guide-2.4.x-10.0.x.md)
-* [migration-from-spray](migration-from-spray.md)
-* [migration-from-old-http-javadsl](migration-from-old-http-javadsl.md)
+* If you are currently using an old version of Akka HTTP or Spray, you should first follow the @link:[Akka HTTP migration guide](https://doc.akka.io/docs/akka-http/10.2/migration-guide/index.html)
```

Comment:
Btw. `@link:` is not strictly required but adds some extra features: https://developer.lightbend.com/docs/paradox/current/directives/linking.html#external-links

Reply:
I've removed the `@link:` in the latest commit.